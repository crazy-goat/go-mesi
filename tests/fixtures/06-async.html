<!DOCTYPE html>
<html>
<head>
    <title>Async request using esi:include</title>
</head>
<body>
This should print hello world: [<esi:include src="http://127.0.0.1:8080/sleep/1/1" />]
This should print hello world: [<esi:include src="http://127.0.0.1:8080/sleep/1/2" />]
This should print hello world: [<esi:include src="http://127.0.0.1:8080/sleep/1/3" />]
This should print hello world: [<esi:include src="http://127.0.0.1:8080/sleep/1/4" />]
This should print hello world: [<esi:include src="http://127.0.0.1:8080/sleep/1/5" />]
</body>
</html>
