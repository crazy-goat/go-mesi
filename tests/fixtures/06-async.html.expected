<!DOCTYPE html>
<html>
<head>
    <title>Async request using esi:include</title>
</head>
<body>
This should print hello world: [1 Waited 1]
This should print hello world: [2 Waited 1]
This should print hello world: [3 Waited 1]
This should print hello world: [4 Waited 1]
This should print hello world: [5 Waited 1]
</body>
</html>
